wellregistry/registry/migrations/0000_create_db_users.py

```diff
@@ -0,0 +1,97 @@
+"""
+migration: initial users and roles
+"""
+from django.db import migrations
+from wellregistry.wellregistry.env import Environment
```

Comment:
@ayan-usgs Could you take a look at this import? I works in pycharm but travis fails to locate it.

Reply:
Have you tried running the tests locally? I suspect the issue is that travis changes directory to wellregistry. If you try to run the command in the project root I wonder what happens?

Reply:
thanks, that make sense. Is there pycharm config to be the same as travis?

Reply:
Not sure I understand the question. I think the solution may be to modify the .travis.yml.

Reply:
out of date.